### ISSUE TITLE
Example for parameters

### ISSUE THREAD

**Issue by nemenemsrouge** [OAI/OpenAPI-Specification#743]
Hello,

Why it is not possible to define example for parameters?

I would like to have something like that

```yaml
- name: "external_id"
  in: "query"
  description: "Filter the result set by **external_id**"
  required: false
  type: "string"
  example: "123-456#789:0.2"
```

but swagger editor says : 

> Not a valid parameter definition

I just don't understand why it is not included into standards specs

**Comment by baileylo** [comment]
Especially since body's that are `type: object` will have properties that can have default values.

**Comment by webron** [comment]
Unfortunately, it's not part of the 2.0 spec, but is included in the 3.0 spec (still WIP atm).
